<figure><img src="https://udemy-images.s3.amazonaws.com:443/redactor/2017-01-03_12-09-06-ca3123e2c06947585b9f09b3854284bf/1%20point.png"></figure>

<p><strong><u>Exercise for reference</u>: </strong></p>









<pre class="prettyprint linenums">d = {"Name": "John", "Surname": "Smith"}
print(d["Smith"])</pre>

























<p><strong><u>Answer</u>: </strong></p>













<p>There is no key <code>Smith </code> in the dictionary. <code>Smith </code> is a value. You want to use <code>Surname</code>  if you want to access <code>Smith</code> :</p><pre class="prettyprint linenums">print(d["Surname"])</pre><p><strong><u>Explanation</u>: </strong><br></p>













<p>A KeyError always means Python could not find a key with the name shown next to KeyError (e.g. <code>Smith</code> ).</p>